docker/Dockerfile

```diff
@@ -0,0 +1,25 @@
+
+FROM openjdk:11.0.2-jre-slim-stretch
+
+COPY ethsigner /opt/ethsigner/
```

Comment:
What gets copied here? ...the dist, the exe or everything?

Reply:
The contents of the dist. So it's as if you unpacked the dist file into /opt/ethsigner